### PR TITLE
#1026 Add checkbox and role summary for Welsh posts

### PR DESCRIPTION
**Pull request by mbrookeswebdev**

### CODE REVIEW

**Review by warrensearle — CHANGES_REQUESTED** [review]
Need to include data on the Show page too 👎

**Comment by mbrookeswebdev** [comment]
> Need to include data on the Show page too 👎

@warrensearle done :)

File: src/views/Exercises/Show/Summary.vue [hunk 1]
```diff
@@ -93,10 +112,16 @@ export default {
     },
     isPublished() {
       return this.exercise.published;
-    }, 
+    },
     canPublish() {
       return this.exercise.progress && this.exercise.progress.vacancySummary;
     },
+    areThereWelshPosts() {
```
Comment (warrensearle) [inline src/views/Exercises/Show/Summary.vue:119]: Ooh we have a filter for this.
In the template you can do
```
{{ exercise.welshPosts | toYesNo }}
```

**Review by warrensearle — CHANGES_REQUESTED** [review]
See comment.

File: src/views/Exercises/Show/Summary.vue (hunk 1 above)
Comment (mbrookeswebdev) [reply src/views/Exercises/Show/Summary.vue]: @warrensearle sorted

**Review by warrensearle — APPROVED** [review]
Marvellous 👍

**Pull request merged**